test/src/units_of_measurement/area_test.dart

```diff
@@ -14,3 +16,30 @@ void _areaUnits() {
     });
   });
 }
+
+void _toString() {
```

Comment:
Each UoM now has smoke-tests for `toString`, but the complex logic in formatting is shared between all UoMs, and is therefore tested independently in `shared_test.dart`.